.github/workflows/ci.yml

```diff
@@ -210,7 +210,7 @@ jobs:
           dfly-executable: dragonfly
           run-only-on-ubuntu-latest: true
           build-folder-name: build
-          filter: ${{ matrix.build-type == 'Release' && '(not slow) and (not dbg_only)' || '(not slow) and (not opt_only)' }}
+          filter: ${{ matrix.build-type == 'Release' && '(not dbg_only)' || '(not slow) and (not opt_only)' }}
```

Comment:
`dbg_only` is not present anywhere in pytests. is it ok?
also maybe add a comment explaining that for non-release builds we run only `non slow` and `not opt_only` builds

Reply:
looks like dbg_only was only for one test in the past and this marker was removed
I will add the comment